pkg/sink/sink.go

```diff
@@ -105,6 +114,13 @@ func (r Sink) HandleEvent(response http.ResponseWriter, request *http.Request) {
 	code := http.StatusAccepted
 	for i := 0; i < len(el.Spec.Triggers); i++ {
 		thiscode := <-result
+		// current take - if someone is doing unauthorized stuff, we abort immediately;
+		// unauthorized should be the final status code vs. the less than comparison
```

Comment:
sounds good to me...

Reply:
same situation here re: https://github.com/tektoncd/triggers/pull/454#discussion_r387717992 @dibyom

Reply:
Again, agreed!